### ISSUE TITLE
ZeroNet shouldn't be re-running coffee on the pre-installed files.

### ISSUE THREAD

**Issue by yurivict** [HelloZeroNet/ZeroNet#477]
I am seeing this error in the log:

```
[2016-05-31 17:35:18,125] DEBUG    - Running: "/usr/local/bin/coffee" --no-header -p "src/Ui/media/lib/ZeroWebsocket.coffee" (Done in 0.25s)
[2016-05-31 17:35:18,305] DEBUG    - Running: "/usr/local/bin/coffee" --no-header -p "src/Ui/media/lib/jquery.csslater.coffee" (Done in 0.18s)
[2016-05-31 17:35:18,306] ERROR    - UiWSGIHandler error: IOError: [Errno 13] Permission denied: 'src/Ui/media/all.js' in UiServer.py line 39 > pywsgi.py line 871 > pywsgi.py line 860 > SidebarPlugin.py line 31 > UiRequest.py line 378 > DebugMedia.py line 122
```
ZeroNet is attempting to overwrite the read-only installed file all.js and fails. It first runs coffee command on the installed file src/Ui/media/lib/ZeroWebsocket.coffee. ZN shouldn't be re-running coffee on the installed files.

**Comment by HelloZeroNet** [comment]
the re-compile executed when the .coffee modification date is newer, than all.js. Maybe adding 1 second window could solve the problem. Can you check the src/Ui/media/lib/ZeroWebsocket.coffee vs src/Ui/media/all.js modification date difference?

**Comment by yurivict** [comment]
Yes, that is it, when I touched all all.js files after installation this problem and #476 went away.

But this is generally a problem when you have ZN installed read-only by the package installer.

You probably want to skip running the coffee compiler on the files under plugins/ and src/ if these all.js files exist and are read-only.
